pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/gp/GPReceivedKit.java

```diff
@@ -79,7 +80,7 @@ public static Optional<KitInfo> receiveKit(String kitLabel, BSPKitDto bspKitQuer
         logger.info("Kit returned has sample id " + bspSampleId);
         return Optional.of(
                 new KitInfo(bspKitQueryResult.getBspCollection(), bspOrganism, "U", bspParticipantId, bspSampleId, bspMaterialType,
-                        bspReceptacleType, ddpInstance.getName(), bspKitQueryResult.getKitTypeName()));
+                        bspReceptacleType, ddpInstance.getName(), bspKitQueryResult.getKitTypeName(), collectionDate));
```

Comment:
Is that change communicated between us and GP? BSP doesn't care for collection date!

Reply:
fixed, we are now returning another object for gp that doesn't have colelctionDate and KitInfo is used only when receiving for both BSP and MERCURY